conf/plants/species_tree.topology.nw

```diff
@@ -244,7 +250,16 @@
           )Anacardiaceae,
          )malvids,
         )rosids,
-        beta_vulgaris,
+        ( 
+         )
```

Comment:
I think this parenthesis is in the wrong direction:
```suggestion
         (
```

Reply:
Thanks for catching that one, fixed in the following commit